parquet-thrift/src/main/java/parquet/thrift/ThriftSchemaConvertVisitor.java

```diff
@@ -56,221 +58,232 @@
  */
 public class ThriftSchemaConvertVisitor implements ThriftType.TypeVisitor {
```

Comment:
indentation is two spaces. This diff is much bigger than it should because of the re-indentation.